B827EBFFFE03284C.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "eui-bcbcbcbcbcbcbcbc",
```

Comment:
This is an invalid gateway ID. It should match the one in the name of the file (e.g.`B827EBFFFE03284C`)

Reply:
Thank you